### DISCUSSION TITLE
allure Reports

### DISCUSSION THREAD

**Discussion by mariamaslam** [mmisty/cypress-allure-adapter#69] [Q&A] [answered]
Hello everyone & @mmisty 

how to add trend graphs in Allure report, how can remove the following from the report:

- allure
- true
- allureCleanResults
- true
- allureSkipCommands
- wrap
- allureResults

<img width="931" alt="Screenshot 2023-11-06 at 1 03 17 PM" src="https://github.com/mmisty/cypress-allure-adapter/assets/47422016/77cefef6-b906-41b6-a481-53dd486fdd8c">

**Comment by mmisty** [comment, marked as answer]
Hi @mariamaslam 
You have two different questions
I'll answer second one for now
 - Environment in Allure report is everything you have in `environment.properties` file. So if you don't want to see env variables in it - do not write them into this file. Writing is done by ` reporter?.writeEnvironmentInfo`. Example:
 ```
         on('before:run', details => {
            reporter?.writeEnvironmentInfo({
               info: {
                  os: details.system.osName,
                  osVersion: details.system.osVersion,
               },
            });
         });
 ```

**Comment by mmisty** [comment]
For the first question the answer is here https://www.google.com/search?q=adding+history+to+allure+report&rlz=1C5CHFA_enRU910RU914&oq=adding+history+to+allure+r&gs_lcrp=EgZjaHJvbWUqBwgBECEYoAEyBggAEEUYOTIHCAEQIRigATIKCAIQIRgWGB0YHtIBCDUyNjFqMGo3qAIAsAIA&sourceid=chrome&ie=UTF-8

https://allurereport.org/docs/gettingstarted-history/

**Comment by mariamaslam** [comment]
Hi @mmisty  i have an other question, since you are expert in cypress and js
Please can you have a look n support in this, i will be grateful
https://github.com/levz0r/gmail-tester/issues/138

**Reply by mmisty** [reply]
Hi @mariamaslam
This is not the first time when you ask to spend my time on something not related to my packages. It seems that you need a consultant. If you want me to be one for you we could discuss a compensation.

**Reply by mariamaslam** [reply]
hey @mmisty no offense im still a learner and exploring new things and new plugins pacakges in cypress, since you are expert in this domain thats why i though maybe you can give it a glance..
Anyways from now on, i will be careful :)
